src/NLog/LoggerImpl.cs

```diff
@@ -96,17 +96,20 @@ internal static void Write([NotNull] Type loggerType, TargetWithFilterChain targ
             }
 
             IList<Filter> prevFilterChain = null;
+            FilterResult? prevDefaultResult = null;
+
             FilterResult prevFilterResult = FilterResult.Neutral;
             for (var t = targets; t != null; t = t.NextInChain)
             {
-                FilterResult result = ReferenceEquals(prevFilterChain, t.FilterChain) ?
-                    prevFilterResult : GetFilterResult(t.FilterChain, logEvent);
+                FilterResult result = ReferenceEquals(prevFilterChain, t.FilterChain) && Equals(prevDefaultResult, t.DefaultResult) ?
```

Comment:
Don't think you need to modify this line. If ReferenceEquals returns true, then everything is identical.

Reply:
but `DefaultResult` isn't in `t.FilterChain`? So i could be another one?

Reply:
Is it possible to have two targets with the same `FilterChain`-list-reference where the DefaultResult is different? The `FilterChain`-list and `DefaultResult` comes from the same LoggingRule-object (Filters + DefaultFilterResult).

Reply:
currently not I think.

OK,thanks, so I can remove 15ef65853eb496859f2778a2db8f3c54733cdf3b from this branch and then it's done.